### PR TITLE
validate html

### PR DESCRIPTION
**Pull request by NullableInt**
To test that we are good boys

### CODE REVIEW

File: .docker/docker-compose.test.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: '3.7'
+
+services:
+  jekyll:
+    command: rake test
```
Comment (asbjornu) [inline .docker/docker-compose.test.yml:5]: Wow, we found a use for Rake! 🎉

**Comment by NullableInt** [comment]
Almost there

File: .docker/docker-compose.test.yml (hunk 1 above)
Comment (NullableInt) [reply .docker/docker-compose.test.yml:5]: Correct! And it almost works as well.

File: Rakefile [hunk 2]
```diff
@@ -19,4 +20,11 @@ task :clean do
   Jekyll::Commands::Clean.process({})
 end
 
+# Test generated output has valid HTML and links.
+task :test do
+  sh "bundle exec jekyll build"
```
Comment (asbjornu) [inline Rakefile:25]: Isn't it better to do `Jekyll::Commands::Build.process(profile: true)` as on line 15?
```suggestion
  Jekyll::Commands::Build.process(profile: true)
```

**Comment by NullableInt** [comment]
> 
> 
> Great work! I think we have a bit too many `Dockerfile`s now, so let's see if we can reduce that in a future PR. This is good to go, though. 👍

Reducing the amount can't wait.

File: Rakefile [hunk 3]
```diff
@@ -19,4 +20,11 @@ task :clean do
   Jekyll::Commands::Clean.process({})
 end
 
+# Test generated output has valid HTML and links.
+task :test do
+  Rake::Task["build"].execute
```
Comment (asbjornu) [inline Rakefile:25]: I think the correct way to do this is:

```suggestion
task :test => :build do
```

Comment (NullableInt) [reply Rakefile:25]: Yes, that also works.

**Pull request merged**